features/app/components/AbstractApp.js

```diff
@@ -22,6 +31,30 @@ export class AbstractApp extends Component {
     }
 
     /**
+     * Init lib-jitsi-meet and create local participant when component is going
+     * to be mounted.
+     *
+     * @inheritdoc
+     */
+    componentWillMount() {
```

Comment:
OK, I tried both ```componentWillMount``` and ```componentDidMount```. If an error is thrown in ```componentWillMount```, then component is not rendered at all, ```componentWillUnmount``` is not called even if we have ```unstable_handleError``` defined (due to another bug). If an error is thrown inside ```componentDidMount```, then component is still rendered and ```componentWillUnmount``` will be called when component is unmounted.